src/vs/workbench/api/common/extHostTesting.ts

```diff
@@ -296,7 +295,7 @@ export class ExtHostTesting extends Disposable implements ExtHostTestingShape {
 	public $publishTestResults(results: ISerializedTestResults[]): void {
 		this.results = Object.freeze(
 			results
-				.map(r => deepFreeze(Convert.TestResults.to(r)))
+				.map(Convert.TestResults.to)
```

Comment:
We apparently cannot deep freeze URIs, since it causes some getters that cache stuff internally to error out